src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -26,11 +26,13 @@
         <service id="routing.loader.xml" class="Symfony\Component\Routing\Loader\XmlFileLoader">
             <tag name="routing.loader" />
             <argument type="service" id="file_locator" />
+            <argument>_controller</argument>
```

Comment:
if `_controller` is going to be the default in the loaders constructors, then this should be removed (same below)